src/main/java/bio/terra/testrunner/common/utils/AuthenticationUtils.java

```diff
@@ -13,7 +13,6 @@
 
 public final class AuthenticationUtils {
   private static volatile GoogleCredentials applicationDefaultCredential;
-  private static volatile GoogleCredentials serviceAccountCredential;
   private static Map<String, GoogleCredentials> delegatedUserCredentials =
```

Comment:
Should we also remove this caching from `getDelegatedUserCredential` and `getApplicationDefaultCredential`?

Reply:
Yes I prefer remove caching these also, will make the change.

Reply:
Great, thanks a lot!